p2p/pex_reactor_test.go

```diff
@@ -180,6 +186,47 @@ func TestPEXReactorAbuseFromPeer(t *testing.T) {
 	assert.True(r.ReachedMaxMsgCountForPeer(peer.NodeInfo().ListenAddr))
 }
 
+func TestPEXReactorUsesSeedsIfNeeded(t *testing.T) {
+	dir, err := ioutil.TempDir("", "pex_reactor")
+	require.Nil(t, err)
+	defer os.RemoveAll(dir) // nolint: errcheck
+
+	book := NewAddrBook(dir+"addrbook.json", false)
```

Comment:
filepath.Join

Reply:
no benefit. I believe `dir` already has `/` at the end